kornia/color/lab.py

```diff
@@ -4,6 +4,7 @@
 import torch.nn as nn
 
 from .xyz import rgb_to_xyz, xyz_to_rgb
+from .rgb import rgb_to_linear_rgb, linear_rgb_to_rgb
```

Comment:
I'm not really liking these function names `rgb_to_linear_rgb, linear_rgb_to_rgb` although `srgb_to_linear_rgb, linear_rgb_to_srgb` would have been more accurate. I stuck with just RGB for consistency. maybe we should mention somewhere in the docs that kornia assumes sRGB as the default RGB format.

Reply:
and what about simply `srgb_to_rgb` <-> `rgb_to_srgb` ?

Reply:
`srgb_to_rgb` might be more confusing because `linear_rgb != rgb`. 
I'm leaving it as it is because we use `srgb==rgb` anyway